clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy

```diff
@@ -122,16 +123,21 @@ class OperationsController {
    * @param id
    */
   @PostMapping("/task/{id}:resume")
-  void resumeTask(@PathVariable("id") String id) {
+  StartOperationResult resumeTask(@PathVariable("id") String id) {
     Task t = taskRepository.get(id);
     if (t == null) {
       throw new NotFoundException("Task not found (id: $id)")
     }
 
-    // TODO(rz): Check if task is failed: Only allow resuming failed tasks
-    // TODO(rz): Lookup saga
+    if (!t.status.retryable) {
+      throw new ConstraintViolationException("Task is not retryable").with {
```

Comment:
I think you could `.also` here instead of `.with` ( and not need the `it` line )

Reply:
`.also` isn't available in Groovy, is it?